java/connecting-to-bigtable/src/main/java/com/example/cloud/bigtable/connecting/BigtableHelper.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START bigtable_connecting_helper]
+
+package com.example.cloud.bigtable.connecting;
+
+import com.google.cloud.bigtable.hbase.BigtableConfiguration;
+import com.google.cloud.bigtable.hbase.BigtableOptionsFactory;
+import java.io.IOException;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.hbase.client.Connection;
+
+public class BigtableHelper {
+
+  public static String projectId;
+  public static String instanceId;
+  public static String appProfileId = "default";
+  public static Connection connection = null;
+
+  public static void main(String... args) {
+    projectId = args[0];  // my-gcp-project-id
+    instanceId = args[1]; // my-bigtable-instance-id
+
+    if (args.length > 2) {
+      // Include the following line if you are using app profiles.
```

Comment:
This is confusing , since it says include the following line vs provide a 3rd argument : eg, my-app-id.
It might suffice to state "If using Cloud Bigtable replication, provide an application profile : 'my-app-id'. If an application profile is not provided, the default is used.

Reply:
There should be separate tags for app_profile vs. not.  FWIW, both should use variants of `Bigtable.connect` as of the upcoming 1.5.0 release.

Reply:
I changed this to just say it will use the default if not provided. From conversations with Misha and Sandy, it seems like we want users to use App Profiles even if they aren't using replication